Mod Configuration Menu/Mods/BG3MCM/ScriptExtender/Lua/Client/IMGUILayer.lua

```diff
@@ -19,7 +19,17 @@
 ---@field private visibilityTriggers table<string, table>
 IMGUILayer = _Class:Create("IMGUILayer", nil, {
     mods = {},
-    visibilityTriggers = {}
+    visibilityTriggers = {},
+    operators = {
+        ["=="] = function(a, b) return a == b end,
+        ["!="] = function(a, b) return a ~= b end,
+        ["<="] = function(a, b) return a <= b end,
+        [">="] = function(a, b) return a >= b end,
+        ["<"] = function(a, b) return a < b end,
+        [">"] = function(a, b) return a > b end,
+        ["isVisible"] = function(a, b) return a == b end
+    },
+    uiElementByName = {}
```

Comment:
I don't like the idea of attaching more lookup stuff to the class... we haven't been needing to do this

Reply:
i write exemple of the requierement of that.